cms/management/commands/import_from_git.py

```diff
@@ -59,6 +65,39 @@ def emit(self, message):
         if not self.quiet:
             self.stdout.write(message)
 
+    def get_thumbor_image_file(self, host, uuid, use_safe_url=False):
+        '''
+        NOTE: To download an image from the Thumbor host we
+        either need a security key for the particular host,
+        or the host must allow unsafe urls.
+        '''
+        if use_safe_url:
+            url = thumbor_image_url(uuid)
+        else:
+            url = urljoin(host, 'unsafe/%s' % uuid)
+
+        response = requests.get(url)
+        if response.status_code == 200:
+            return (
+                ContentFile(response.content),
+                response.headers['Content-Type'])
+        return None, None
+
+    def set_image_field(self, eg_obj, db_obj, field_name):
+        uuid = getattr(eg_obj, field_name)
+        host = getattr(eg_obj, '%s_host' % field_name)
+        if None in (uuid, host):
+            return
+
+        file_obj, content_type = self.get_thumbor_image_file(host, uuid)
```

Comment:
I think it can be done, but it then all apps must use the same Thumbor server. Otherwise the app won't be able to generate a valid signed url for the image.